### DISCUSSION TITLE
Strange DMCA issue

### DISCUSSION THREAD

**Discussion by kuro68k** [community/community#39184] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I had an email about a DMCA claim against one of my repositories. The URL is

https://github.com/github/dmca/blob/master/2022/11/2022-11-10-mediatek.md

This URL is 404. The repo is also gone.

The repo contained open source code from a Chinese company, that was only published after intervention by Naomi Wu.

What is going on here? It appears to be a claim by Mediatek. If it is genuine I'll remove the code in question. There is gigabytes of it, I doubt they are claiming all of it.

I'm also worried about "red flags" on my account.

Advice appreciated.

**Comment by david-engelmann** [comment]
[Github has a great DMCA counter notice guide, check it out here](https://support.github.com/contact/dmca-counter-notice)! They will be able to clear up any confusion!

**Comment by kuro68k** [comment]
Thanks, but since I can't read the original notice I don't know what they are claiming. I can't evaluate the claim.

**Comment by david-engelmann** [comment]
If you reach out to support@github.com they will be able to further assist you. They will have the claim available

**Comment by airtower-luna** [comment]
> Thanks, but since I can't read the original notice I don't know what they are claiming. I can't evaluate the claim.

I think the Markdown file you linked is probably the entire claim. They make the sworn statement claiming copyright, and under DMCA GitHub has to take certain actions (like removing the code). :confused: Can't hurt to ask support, though.

The key question is whether or not the claim was rightful (I can't answer that!). If it was not, filing a counter notice makes sense. You might want to get legal advice on that, and a lawyer could also help you with filing a counter-notice if you choose to. There are some organizations that aim to help Free Software developers with such problems, the [Software Freedom Law Center (SFLC)](https://softwarefreedom.org/) is the first that comes to my mind.

**Reply by david-engelmann** [reply]
Great expansion! Don't file a counter notice unless you think it's been wrongly flagged

**Reply by kuro68k** [reply]
Okay, the link is working now. That was weird. Earlier it was 404.

Anyway, yes, they claim the entire repo which is ridiculous. It's not even my code though, so I'm not going to waste my own money getting legal advice. Interestingly that means that UMIDIGI are distributing Mediatek's copyrighted code because they can't be bothered with proper GPL compliance and just dump everything they have on you.

This is an interesting new vulnerability. Say you want some proprietary code but don't want to sign an NDA, just hunt around for someone using some GPL code along side it and ask them for a copy.

I'll think about if I want to contest it. Shame Github is US based, I live in Europe and the DMCA doesn't apply to me. The Mediatek code is still on Github too: https://github.com/UMIDIGI-Official/UMIDIGI-F2-Kernel-Source-Code

**Reply by airtower-luna** [reply]
> Anyway, yes, they claim the entire repo which is ridiculous. It's not even my code though, so I'm not going to waste my own money getting legal advice.

Totally understandable, which is why I pointed out organizations like the SFLC. If they take up the case you shouldn't need to hire a lawyer. I haven't been in that situation myself, but the point is that they want to support Free Software, so if their lawyers conclude the claim was fraudulent, they'll probably be interested in fighting it. :wink:
